packages/manager/src/features/Linodes/LinodesCreate/SelectPlanPanel/KubernetesPlanSelection.tsx

```diff
@@ -1,15 +1,23 @@
 import * as React from 'react';
 import Grid from '@mui/material/Unstable_Grid2';
+import { useTheme, styled } from '@mui/material/styles';
 import Button from 'src/components/Button';
-import classNames from 'classnames';
 import { EnhancedNumberInput } from 'src/components/EnhancedNumberInput/EnhancedNumberInput';
 import Hidden from 'src/components/core/Hidden';
 import SelectionCard from 'src/components/SelectionCard';
 import { TableCell } from 'src/components/TableCell';
-import { TableRow } from 'src/components/TableRow';
 import { convertMegabytesTo } from 'src/utilities/unitConversions';
 import { ExtendedType } from 'src/utilities/extendType';
-import { useSelectPlanQuantityStyles } from './styles/kubernetesPlansPanelStyles';
+import { StyledDisabledTableRow } from './styles';
+
+const StyledInputOuter = styled('div')(({ theme }) => ({
```

Comment:
let's always add a `label` on Styled components for easier debugging

```suggestion
const StyledInputOuter = styled('div', { label:  'StyledInputOuter'})(({ theme }) => ({
```